index.html

```diff
@@ -29,5 +29,142 @@ <h1>Charlie Giorgania</h1>
+          <span>See this project</span>
+          <div class="works-arrow"></div>
+      </div>
+      </a>
+    </div>
+    <img class="works-img" src="./Media/work-img-2.svg" alt="Man typing on a laptop">
+    <div class="works-box">
+      <h3>Project name goes here</h3>
+      <div class="lang-container">
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
+      <a href="#" rel="noopener noreferrer" class="works-link">
+        <div class="works-button">
+          <span>See this project</span>
+          <div class="works-arrow"></div>
+      </div>
+      </a>
+    </div><img class="works-img" src="./Media/work-img-3.svg" alt="Man typing on a laptop">
+    <div class="works-box">
+      <h3>Project name goes here</h3>
+      <div class="lang-container">
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
+      <a href="#" rel="noopener noreferrer" class="works-link">
+        <div class="works-button">
+          <span>See this project</span>
+          <div class="works-arrow"></div>
+      </div>
+      </a>
+    </div><img class="works-img" src="./Media/work-img-4.svg" alt="Man typing on a laptop">
+    <div class="works-box">
+      <h3>Project name goes here</h3>
+      <div class="lang-container">
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
+      <a href="#" rel="noopener noreferrer" class="works-link">
+        <div class="works-button">
+          <span>See this project</span>
+          <div class="works-arrow"></div>
+      </div>
+      </a>
+    </div><img class="works-img" src="./Media/work-img-5.svg" alt="Man typing on a laptop">
+    <div class="works-box">
+      <h3>Project name goes here</h3>
+      <div class="lang-container">
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
+      <a href="#" rel="noopener noreferrer" class="works-link">
+        <div class="works-button">
+          <span>See this project</span>
+          <div class="works-arrow"></div>
+      </div>
+      </a>
+    </div>
+    <img class="works-img" src="./Media/work-img-6.svg" alt="Man typing on a laptop">
+    <div class="works-box">
+      <h3>Project name goes here</h3>
+      <div class="lang-container">
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
+      <a href="#" rel="noopener noreferrer" class="works-link">
+        <div class="works-button">
+          <span>See this project</span>
+          <div class="works-arrow"></div>
+      </div>
+      </a>
+    </div>
```

Comment:
- [ ]The list of projects should use a` ul tag` to group them, and `li` per project

Reply:
Satisfied.